docs/documentation/_includes/reference/cli/werf_helm_show_all.md

```diff
@@ -5,7 +5,7 @@
 {% endif %}
 
 This command inspects a chart (directory, file, or URL) and displays all its content
-(values.yaml, Chart.yaml, README)
+(values.yaml, Charts.yaml, README)
```

Comment:
This change is unrelated to my work, hence it's in a distinct commit before the spelling fixes. Anyone touching this file (and another) will trip on CI failures.